tox.ini

```diff
@@ -86,3 +86,10 @@ skip_install = true
 commands =
     black --config=.black {toxinidir}
     flake8
+
+[testenv:clean]
+description = Remove all that tox stuff
+allowlist_externals = rm
```

Comment:
For several reasons, this makes my heart rate jump.

Reply:
I'm glad that my coding can excite you :-)